lemarche/tenders/models.py

```diff
@@ -78,7 +107,46 @@ def is_live(self):
         return self.sent().filter(deadline_date__gte=datetime.today())
 
     def has_amount(self):
-        return self.filter(Q(amount__isnull=False) | Q(amount_exact__isnull=False))
+        return self.filter(Q(amount__isnull=False) | Q(amount_exact__isnull=False)).annotate(
+            has_amount_exact=Case(
+                When(amount_exact__isnull=False, then=Value(True)), default=Value(False), output_field=BooleanField()
+            )
+        )
+
+    def filter_by_amount(self, amount: int, operation: str = "gte"):
```

Comment:
```suggestion
    def filter_by_amount_exact(self, amount: int, operation: str = "gte"):
```
pour clarifier, car sinon on pense qu'on doit passer un range (comme ailleurs)